.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+<!--
```

Comment:
whenever someone opens a PR in the docs repo, this will be in the PR description